integration-test/pages/login-poor-form.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Login form</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+<p><a href="../index.html">[Home]</a></p>
+
+<p id="demo"></p>
+
+<div class="dialog">
+  <div id="login-form">
+    <label for="email">Email</label>
+    <input id="email" type="email">
+    <label for="password">Password</label>
+    <input id="password" type="password">
+  </div>
+</div>
+<div class="fixed">
+  <div role="button" class="button">Log in</div>
+  <a href="#" role="button">Sign up</a>
+</div>
```

Comment:
This is the web in 2022 — still! This comes from Twitter (worst markup ever), but Google is not much different and Reddit has a similar off-the-form button without the [`form` attribute](dfile:///Users/gsv/Library/Application%20Support/Dash/DocSets/HTML/HTML.docset/Contents/Resources/Documents/developer.mozilla.org/en-US/docs/Web/HTML/Element/button.html#attr-form). 🤦‍♂️